clustoapi/apps/attribute.py

```diff
@@ -21,6 +21,61 @@
 app.config['source_module'] = __name__
 
 
+def _write_attrs(method, name, **kwargs):
+    """
+Helper method for reduced code between POST and PUT.
+Returns a response for the methods calling it.
+"""
+    if method == 'set_attr':
```

Comment:
if you're going to use a common _write_attrs() method, ``set`` and ``add`` is enough for method name, don't use the _attrs suffix